components/menu-items/style.scss

```diff
@@ -1,9 +1,9 @@
-.components-choice-menu {
+.components-menu-items-group {
```

Comment:
This should be `.components-menu-items__group`

https://github.com/WordPress/gutenberg/blob/master/docs/coding-guidelines.md#css

Reply:
Updating

Reply:
Opened #5218 to address it.